packages/badge/package.json

```diff
@@ -8,7 +8,7 @@
   "kata-kit:src": "./src/index.ts",
   "repository": "https://github.com/kata-ai/kata-kit.git",
   "files": [
-    "./lib"
+    "**/lib/*"
```

Comment:
This would be better if we use `"lib"` instead, without the asterisks and slashes.

Also wouldn't this glob pattern mean that subfolders inside `lib` won't be included in the package?

Reply:
yep, you're right. will change that